docs/data/migration/migration-pickers-v7/migration-pickers-v7.md

```diff
@@ -344,6 +344,16 @@ This change causes a few breaking changes:
 
 - The button to render a single year is no longer wrapped in a `<div />`, the spacing are instead defined inside the `root` slot of the Year Calendar.
 
+### Update default `closeOnSelect` and Action Bar `actions` values
+
+The default value of the `closeOnSelect` prop has been updated to `false` for all Picker components with complex views.
+Only `<DesktopDatePicker />` and `<DesktopDateRangePicker />` have `closeOnSelect` set to `true` by default.
+
+This change goes hand in hand with the new default `actions` prop value for the `<PickersActionBar />` component.
+The default value of the `actions` prop has been updated to `['cancel', 'accept']` for all Picker components with complex views, except for `<DesktopDatePicker />` and `<DesktopDateRangePicker />`.
```

Comment:
I would remove the concept of "complex" value which is very subjectif and instead list the pickers.

Reply:
Or use 'advanced'?

Reply:
Same, what is an advanced picker?

Reply:
Good question... 😅

Reply:
What do you think about simplifying to:
```suggestion
The default value of the `actions` prop has been updated to `['cancel', 'accept']` for all Picker components, except `<DesktopDatePicker />` and `<DesktopDateRangePicker />`.
```